Nodejs/Product/TestAdapter/TestFrameworkFactory.cs

```diff
@@ -91,18 +91,6 @@ private static (string, string)? GetTestFrameworkAndFilePath(Project project, Pr
                 fileAbsolutePath = CommonUtils.GetAbsoluteFilePath(projectRoot, projectItem.EvaluatedInclude);
             }
 
-            // Check if file is a typecript file. If so, get the javascript file. The javascript file needs to be in the same path and name.
-            // It doesn't work with bundlers or minimizers. Also, project needs to be build in order to have the js file created.
-            var typeScriptTest = TypeScriptHelpers.IsTypeScriptFile(fileAbsolutePath);
```

Comment:
@armanio123 can you confirm to me that all this is doing is getting the .js and why that was a decision back then? I'm passing the .ts and .tsx now, do you think removing this code will cause other issues?

Reply:
I think this will affect other test frameworks that are not Jest as they depend of this for typescript support. As an option, we might be able to filter out for Jest only.